adding-flash.md

```diff
@@ -2,13 +2,38 @@
 
 Flash is a hash that is generated in one controller action, and is accessible only in the *next* controller action. It is used only to contain messages to the user, not any other sort of data.
 
-By convention, there are two types of flash messages: alerts and notices. Alerts are used for warnings and errors; notices are used for everything else.
+Think of flash as a session that only lasts for one request.
+
+By convention, there are two types of flash messages: alerts and notices.
+- Alerts are used for warnings and errors
+- Notices are used for everything else
 
 ## 1. Replace every `puts` with the appropriate `flash`
 
-Check for `puts` statements in the Application, Sessions, and Users controllers. For instance:
+Check for `puts` statements in the Application, Sessions and Users controllers. For example...
+
+```rb
+# app/controllers/sessions_controller.rb
 
-### [session#create](https://github.com/ga-wdi-exercises/tunr_rails_users/pull/4/files#diff-d5241d488259f32ecbe2f636133e5ddaR12)
+def create
```

Comment:
I would even put a little note here recommending typing it out instead of copy/pasting